package.json

```diff
@@ -51,27 +51,33 @@
       "properties": {
         "alloglot.activateCommand": {
           "type": "string",
-          "description": "A shell command to run on activation. The command will run asynchronously. It will be killed (if it's still running) on deactivation."
+          "description": "A shell command to run on activation. The command will run asynchronously. It will be killed (if it's still running) on deactivation.",
+          "default": null
         },
         "alloglot.deactivateCommand": {
           "type": "string",
-          "description": "A shell command to run on deactivation."
+          "description": "A shell command to run on deactivation.",
+          "default": null
         },
         "alloglot.verboseOutput": {
           "type": "boolean",
-          "description": "If `true`, Alloglot will log more output."
+          "description": "If `true`, Alloglot will log more output.",
+          "default": null
```

Comment:
The issue preventing the fallback from being used is that VS Code defaults boolean config options to `false` if the package.json doesn't specify a different default.

Reply:
:argh: